src/main/java/org/gridsuite/explore/server/ExploreController.java

```diff
@@ -242,6 +242,16 @@ public ResponseEntity<Void> deleteElement(@PathVariable("elementUuid") UUID elem
         return ResponseEntity.ok().build();
     }
 
+    @DeleteMapping(value = "/explore/elements/{directoryUuid}/delete-stashed")
+    @Operation(summary = "Remove directories/elements")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "Filter creation request delegated to filter server")})
+    public ResponseEntity<Void> deleteElements(@RequestParam("ids") List<UUID> elementsUuid,
+                                               @RequestHeader("userId") String userId,
+                                               @PathVariable String directoryUuid) {
```

Comment:
This variable is not used

Reply:
Yes, but we must have an uuid at this position or the gateway will reject our fetch with 403 status